Networking/NetworkingTests/Mapper/UserMapperTests.swift

```diff
@@ -0,0 +1,42 @@
+import XCTest
+@testable import Networking
+
+
+/// UserMapper Unit Tests
+///
+final class UserMapperTests: XCTestCase {
+
+    private let testSiteID: Int64 = 123
+
+    func test_User_fields_are_properly_parsed() {
+        guard let user = mapUserFromMockResponse() else {
+            XCTFail()
+            return
+        }
+
+        XCTAssertEqual(user.localID, 1)
+        XCTAssertEqual(user.username, "test-username")
+        XCTAssertEqual(user.firstName, "Test")
+        XCTAssertEqual(user.lastName, "User")
+        XCTAssertEqual(user.email, "example@example.blog")
+        XCTAssertEqual(user.nickname, "User")
```

Comment:
This won't fail on mistake if you map `lastName` and `nickname` with same data field 🙂
Since we're testing mapping here I suggest unique values.

Reply:
Good catch! Will update this shortly.

Reply:
Fixed!